### PR TITLE
Transactions

### PR DESCRIPTION
**Pull request by tsohleDev**
**Query with transactions**

- Add a column to the table
- Add more data
- Make queries in form of transactions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Vet Clinic] <a name="about-project"></a>
+
+> Vet clinic is a tutorial on how to create a relational database in SQL, where we create a vet clinic's database
```
Comment (danmainah) [inline README.md]: - [x] @tsohleDev kindly remove the guideline statements, they are supposed to only guide you as you create the readme file

File: data.sql [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg, species)
+VALUES 
+  (4,'Charmander', '2020-02-08', 0, false, 11, 'lizard'),
+  (6,'Plantmon', '2021-11-15', 2, true, 5.7, 'plant'),
+  (7,'Squirtle', '1993-04-02', 3, false, 12.13, 'turtle'),
+  (8,'Angemon', '2005-06-12', 1, true, 45, 'angel'),
+  (9,'Boarmon', '2005-06-07', 7, true, 20.4, 'boar'),
+  (10,'Blossom', '1998-10-13', 3, true, 17, 'flower'),
+  (11,'Ditto', '2022-05-14', 4, true, 22, 'transformer');
+
```
Comment (danmainah) [inline data.sql]: - [x] @tsohleDev kindly ensure you have only data provided in the project, remove the species data, and also the weights are correct, some are supposed to be negative.

File: queries.sql [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+
+/*Part Two*/
+-- UPDATE SPECIES WITH ROLLBACK
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
```
Comment (danmainah) [inline queries.sql]: - [x] @tsohleDev kindly add code to  verify that change was made before and after rollback eg ``SELECT species from animals;``

File: queries.sql [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+COMMIT;
+
```
Comment (danmainah) [inline queries.sql]: - [x]  @tsohleDev kindly add code to verify that the change was made before and after the commit.

File: queries.sql [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+-- DELETE ALL RECORDS AND ROLLBACK
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
+-- UPDATE ANIMALS
```
Comment (danmainah) [inline queries.sql]: - [x]  @tsohleDev kindly add code to verify that the change was made before and after the rollback

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @tsohleDev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️

- [ ] Kindly add screenshots of the queries results to the pull request description, and kindly ensure the result are correct

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (tsohleDev) [reply README.md]: I see, thank you for the info

File: queries.sql (hunk 3 above)
Comment (tsohleDev) [reply queries.sql]: yes indeed

File: README.md [hunk 6]
```diff
@@ -0,0 +1,196 @@
+# curriculum-databases-projects-template
+
+> This template should be used for database related projects at Microverse.
+> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
```
Comment (afanimartin) [inline README.md]: - Remove this guidelines as suggested. Add content relevant to the section instead. Do the same to all sections.

File: queries.sql (hunk 3 above)
Comment (afanimartin) [reply queries.sql]: - Please make the requested change. Run a SELECT query after the rollback

File: queries.sql (hunk 5 above)
Comment (afanimartin) [reply queries.sql]: - Please make the requested change. Run a SELECT query after the rollback

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @tsohleDev, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights 👍🏿 👍🏿 👍🏿 
- Prompt communication with reviewers

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 5 above)
Comment (tsohleDev) [reply queries.sql]: I actually did, you just looking at the outdated part

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @tsohleDev,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## To Highlight

✅ Descriptive pull request message
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**